packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -2476,34 +2517,41 @@ export class WalletService {
   }
 
   _validateAddr(wallet, inaddr, opts) {
-    const A = Bitcore_[wallet.coin].Address;
+    if (wallet.coin == 'eth') {
+      try {
+        Validation.validateAddress(
+          wallet.coin.toUpperCase(),
+          'mainnet',
```

Comment:
can we do wallet.network? For ETH it results in the same derivation path for all networks.